src/pages/index.js

```diff
@@ -93,12 +119,16 @@ function handlePlaceCreateForm(data) {
     link: data.link,
   };
   placeCreateFormValidator.disableSubmit();
```

Comment:
`  placeCreateFormValidator.disableSubmit();` is not needed here. You have it in `then`